# Definition of ROS service LocSetMappingActiveSrv for sick localization
# Activates / deactivates mapping while localizing.
# Example call (ROS1):
# rosservice call LocSetMappingActiveSrv "{active: 1}"
# Example call (ROS2):
# ros2 service call LocSetMappingActiveSrv sick_lidar_localization/srv/LocSetMappingActiveSrv "{active: 1}"
# 

# 
# Request (input)
# 

bool active # Flag indicating whether it is desired to activate or deactivate mapping while localizing. {0 (false), 1 (true)} 

---

# 
# Response (output)
# 

bool success # true: success response received from localization controller, false: service failed (timeout or error status from controller)
